share/doc/DEVELOPERS_GUIDE.md

```diff
@@ -185,6 +190,12 @@ make test
 
 Ensure you have an up to date version of cmake or use cmake3 
 
+### Testing Across Repos ###
+
+There may come a case where changes have to be committed across more than one repo at the same time. Attempting to merge one repo at a time will cause the continuous integration to fail as changes in the other repos will not be pulled in. To do this correctly the following steps should be taken:
+
+1. Each repo that is changed must have its own branch, and the branches must have the same name.
+
```

Comment:
Ok, as we don't build branches anymore, the whole "same name" thing doesn't matter anymore.

To test across multiple repos you need to do something like in `votca/votca`:
```
git checkout <base_branch>
git submodule update
git checkout -b <some_ descriptive_branch_name>
git submodule foreach git remote update
git -C <module1> checkout <sha_or_branch_of_module1_to_test>
git -C <module2> checkout <sha_or_branch_of_module2_to_test>
git add <module1> <module2>
git commit -m "test <module1> with <module2>"
git push origin <some_ descriptive_branch_name>
```
and then create a pull request. Once the pull requests passes, merge it and **then** merge the individual PRs in `module1` and `module2`.

Reply:
base_branch - refers to the master or stable branch?

Reply:
And what is the purpose of calling `git submodule update` ? Why not 

    git pull origin <base_branch>
    git reset --hard